charts/spire/charts/spire-server/values.yaml

```diff
@@ -104,6 +104,16 @@ upstreamAuthority:
         certificate: ""
         key: ""
         bundle: ""
+  certManager:
+    enabled: false
+    rbac:
+      create: true
+    issuer_name: "spire-ca"
+    issuer_kind: "Issuer"
+    issuer_group: "cert-manager.io"
```

Comment:
Will there be any usecase where the issuer_kind and issuer_group need to be changed? If not I would leave it out.

Reply:
Not sure. I can't think of one off hand, but was just making it as configurable as the plugin is in spire. I figure if they made it configurable there, they must have had a reason. But I could be convinced otherwise.

If you still want me to pull it, please let me know and I'll do so.

Reply:
@faisal-memon what is your take on this? Do you have more background info on why this is configurable on the plugin?

My take would be to hardcode those values in the chart. But I might be missing the usecase of this.

Reply:
oh... I know of at least one reason....

issuer_kind can be either Issuer or ClusterIssuer.

Reply:
Actually, I can think of a common one that users may need after re-review:
issuer_kind: "ClusterIssuer"